### PR TITLE
zh-translation: /docs/setup/install/operator/index.md

### PR DESCRIPTION
**Pull request by idefav**
ref: https://github.com/servicemesher/istio-official-translation/issues/1287

### CODE REVIEW

File: content/zh/docs/setup/install/operator/index.md [hunk 1]
```diff
@@ -1,58 +1,50 @@
...
+该命令行工具具有用户输入校验，可以防止错误的安装和自定义选项。
 
-Using these instructions, you can select any one of Istio's built-in
-[configuration profiles](/docs/setup/additional-setup/config-profiles/)
-and then further customize the configuration for your specific needs.
+使用这些说明，您可以选择 Istio 的任何内置组件
+[配置文件](/zh/docs/setup/additional-setup/config-profiles/) 然后根据您的特定需求进一步自定义配置。
 
-## Prerequisites
+## 先决条件
```
Comment (rootsongjc) [inline content/zh/docs/setup/install/operator/index.md]: 所有的标题需要添加锚点

File: content/zh/docs/setup/install/operator/index.md [hunk 2]
```diff
@@ -152,95 +142,87 @@ k8s:
...
 {{< /text >}}
 
-## Customizing the configuration
+## 定制配置
 
-In addition to installing any of Istio's built-in
-[configuration profiles](/docs/setup/additional-setup/config-profiles/),
-`istioctl manifest` provides a complete API for customizing the configuration.
+除了安装 Istio 的任何内置组件 [配置文件](/zh/docs/setup/additional-setup/config-profiles/)，
+`istioctl manifest` 提供了用于自定义配置的完整API。
```
Comment (rootsongjc) [inline content/zh/docs/setup/install/operator/index.md]: 英文和中文间要加空格。

File: content/zh/docs/setup/install/operator/index.md [hunk 3]
```diff
@@ -398,13 +376,12 @@ spec:
...
-namespaces and enablement settings. The Istio community recommends using the `IstioControlPlane` API as it is more
-consistent, is validated, and follows the [community graduation process](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE-CHECKLIST.md#feature-lifecycle-checklist).
+一些参数将在 Helm 和 `IstioControlPlane` API 中暂时存在，包括 Kubernetes 资源，
+命名空间和启用设置。 Istio 社区建议使用 `IstioControlPlane` API，因为它更专一，经过验证并遵循[社区毕业流程](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE-CHECKLIST.md#feature-lifecycle-checklist)。
 
-## Uninstall Istio
+## 卸载 Istio
 
-To uninstall Istio, run the following command:
+可以使用以下命令来卸载 Istio
```
Comment (rootsongjc) [inline content/zh/docs/setup/install/operator/index.md]: 漏了冒号

**Comment by jakeslee** [comment]
@rootsongjc: The directory of file has been renamed from `/content/en/docs/setup/install/operator` to `/content/en/docs/setup/install/istioctl`. The content has been updated too. (ref: #5390)
These changes cause istio.io lint failed in PR #5385.
@idefav I think you can push a commit to fix this.

File: content/zh/docs/setup/install/operator/index.md (hunk 1 above)
Comment (idefav) [reply content/zh/docs/setup/install/operator/index.md]: 已经全部添加英文锚点

File: content/zh/docs/setup/install/operator/index.md (hunk 2 above)
Comment (idefav) [reply content/zh/docs/setup/install/operator/index.md]: 已经修改

File: content/zh/docs/setup/install/operator/index.md (hunk 3 above)
Comment (idefav) [reply content/zh/docs/setup/install/operator/index.md]: 已经修改

File: .spelling [hunk 4]
```diff
@@ -70,6 +70,7 @@ analytics
 ANDed
```
Comment (rootsongjc) [inline .spelling]: 请不要修改该文件。

Comment (idefav) [reply .spelling]: 已经还原

**Comment by idefav** [comment]
@rootsongjc @jakeslee I have updated the opeator to istioctl directory and update the link from file helm/index.md

**Pull request merged**